pkg/artifactory/resource_artifactory_remote_repository.go

```diff
@@ -339,6 +371,15 @@ func packRemoteRepo(repo *v1.RemoteRepository, d *schema.ResourceData) error {
 	logErr(d.Set("vcs_git_provider", repo.VcsGitProvider))
 	logErr(d.Set("vcs_type", repo.VcsType))
 	logErr(d.Set("xray_index", repo.XrayIndex))
+	if repo.Nuget != nil {
+		logErr(d.Set("nuget", []interface{}{
+			map[string]interface{}{
+				"feed_context_path":     repo.Nuget.FeedContextPath,
```

Comment:
Should this be actual strings and not *string.

Reply:
`repo.Nuget.FeedContextPath` returns string pointers
https://github.com/atlassian/go-artifactory/blob/v2/artifactory/v1/repositories.go#L157-L159